exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,127 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "Students' gardens are assigned in alphabetical order.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
```

Comment:
Ah, I was meaning to refer only to the list of the twelve default students...

But now that you bring it up, yes, "Students' gardens are assigned in alphabetical order." is also mentioned in the description! So I would also think that should be removed.

If there is to be clarification for #659, I would suggest it go in the description since the README is where students will likely go to look for instructions for how to do this exercise. It wouldn't be visible to students if it were placed in this canonical-data.json file. Since this PR is just dealing with the canonical data, we don't need to deal with this now, unless you would like to add another commit to this PR or a separate PR.